Docker/entrypoint.sh

```diff
@@ -34,13 +34,8 @@ if [ -n "$OIDC_ENABLED" ] && [ "$OIDC_ENABLED" -ne 0 ]; then
 fi
 
 if [ -n "$CRON_MIN" ]; then
-	(
-		echo "export TZ=$TZ"
-		echo "export COPY_LOG_TO_SYSLOG=$COPY_LOG_TO_SYSLOG"
-		echo "export COPY_SYSLOG_TO_STDERR=$COPY_SYSLOG_TO_STDERR"
-		echo "export FRESHRSS_ENV=$FRESHRSS_ENV"
-		echo "export DATA_PATH=$DATA_PATH"
-	) >/var/www/FreshRSS/Docker/env.txt
+	# shellcheck disable=SC2002
+	cat /proc/self/environ | tr '\0' '\n' | grep -vE '^(HOME|PATH|PWD|SHLVL|TERM|_)=' | sort -u | sed 's/^/export /' >/var/www/FreshRSS/Docker/env.txt
 	sed </etc/crontab.freshrss.default \
```

Comment:
This line is literally the syntax in question so something doesn't smell quite right. ;-)

Reply:
Previous line, with cat

Reply:
Yes, but this is the exact same situation. You could put a useless cat there.

So I'd assume it's something about `tr` then, not about Ash, though odd either way.

Reply:
I think the difference is that in this PR, we access a process-dependent file